### PR TITLE
Landing page for FlowFuse Dashboard

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

Created the landing page for FlowFuse Dashboard and added it as an item under the `resources` menu.

The original issue requested this to be the URL `/node-red-dashboard2/` but since we re-named it, I changed the URL to `/flowfuse-dashboard/`

As per @Hasmin-AC request, I'll change the image to be more [these](https://dashboard.flowfuse.com/nodes/widgets.html). I'm leaving that for a future iteration.

## Related Issue(s)

[#2131](https://github.com/FlowFuse/website/issues/2131)

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by joepavitt** [comment]
I do still need to actually tie up the migration service properly too, so I can do that first thing on Tuesday - possibly on Sunday.

**Comment by Hasmin-AC** [comment]
Thanks @Yndira-E I would like to suggest listing this under the products menu instead. @ZJvandeWeg and @joepavitt WDYT?

**Comment by joepavitt** [comment]
I think that's a great idea

File: src/flowfuse-dashboard.njk [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+layout: nohero
+meta:
+  title: FlowFuse Dashboard
+  description: Discover FlowFuse Dashboard for Node-RED 2.0 - the ultimate tool for creating visually appealing and functional UIs in minutes. Empower your development with drag-and-drop simplicity, enhanced user experiences, and a growing ecosystem of widgets. Seamlessly migrate from Dashboard 1.0 and explore comprehensive learning resources. Contact our experts for tailored solutions.
```
Comment (Hasmin-AC) [inline src/flowfuse-dashboard.njk:5]:
```suggestion
  description: Create visually appealing and functional UIs in minutes with FlowFuse Dashboard for Node-RED, the open-source low-code platform for connecting data endpoints
```

File: src/flowfuse-dashboard.njk [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
```
Comment (ZJvandeWeg) [inline src/flowfuse-dashboard.njk:1]: Why not `/src/product/dashboard.njk`?

Comment (joepavitt) [reply src/flowfuse-dashboard.njk:1]: I think that's @Hasmin-AC's point above, I agree that it makes sense to put it there.

File: src/_includes/layouts/base.njk [hunk 3]
```diff
@@ -216,15 +216,16 @@ eleventyComputed:
...
-                            <ul class="sm:grid sm:grid-flow-col sm:grid-rows-4 sm:grid-cols-1 sm:pr-9">
+                            <ul class="sm:grid sm:grid-flow-col sm:grid-rows-5 sm:grid-cols-1 sm:pr-9">
                                 {% navoption "Blog", "/blog/", 1, "newspaper" %}{% endnavoption %}
                                 {% navoption "Webinars", "/webinars/", 1, "screen" %}{% endnavoption %}
                                 {% navoption "Github", "https://github.com/FlowFuse/flowfuse", 1, "github", true %}{% endnavoption %}
                                 {% navoption "Docs", "/docs/", 1, "document-text" %}{% endnavoption %}
                                 {% navoption "Support forums", "https://discourse.nodered.org/c/vendors/flowfuse/24/", 1, "chat-bubble-left-right-sm" %}{% endnavoption %}
                                 {% navoption "Customer Stories", "/customer-stories/", 1, "presentation" %}{% endnavoption %}
-                                {% navoption "Node-RED", "/node-red/", 1, "info" %}{% endnavoption %}
+                                {% navoption "Node-RED", "/node-red/", 1, "nr-icon" %}{% endnavoption %}
```
Comment (ZJvandeWeg) [inline src/_includes/layouts/base.njk:226]: Thanks

File: src/_includes/migration.njk [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+            });
+        }
+    }
+
+    function dragOverHandler(event) {
+        event.preventDefault();
+    }
+
+    const PROD = '968a9ab6-3dd8-45b2-991c-3f055dc18787'
+    const TEST = '1ecbeb84-36a6-4077-b6c8-3f4a5b06bca0'
```
Comment (ZJvandeWeg) [inline src/_includes/migration.njk:111]: I don't think we should ever put secrets in code like this, but expose it through environment variables.

Can you please cycle these ASAP?

Comment (joepavitt) [reply src/_includes/migration.njk:111]: sure, it does still live in plain text on our website though

File: src/flowfuse-dashboard.njk (hunk 2 above)
Comment (Yndira-E) [reply src/flowfuse-dashboard.njk]: Done!

File: src/_includes/migration.njk (hunk 4 above)
Comment (ZJvandeWeg) [reply src/_includes/migration.njk]: I thought this was something else, my bad

**Review by ZJvandeWeg — APPROVED** [review]

**Comment by joepavitt** [comment]
> I do still need to actually tie up the migration service properly too, so I can do that first thing on Tuesday - possibly on Sunday.

Can confirm this is resolved. the migration service is live

**Review by Hasmin-AC — APPROVED** [review]

**Pull request merged**